app/views/profiles/_profile.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="medium-12 columns">
```

Comment:
This isn't needed, since the parent node is already defining how these contents should be laid out.